src/qt/guiutil.cpp

```diff
@@ -759,14 +759,14 @@ QString formatNiceTimeOffset(qint64 secs)
 
 QString formatBytes(uint64_t bytes)
 {
-    if(bytes < 1024)
+    if(bytes < 1000)
         return QObject::tr("%1 B").arg(bytes);
-    if(bytes < 1024 * 1024)
-        return QObject::tr("%1 KB").arg(bytes / 1024);
-    if(bytes < 1024 * 1024 * 1024)
-        return QObject::tr("%1 MB").arg(bytes / 1024 / 1024);
+    if(bytes < 1000 * 1000)
+        return QObject::tr("%1 kB").arg(bytes / 1000);
```

Comment:
@hebasto: This seems like a good place for some context to translators. For example, look at the French translation below.
![Screen Shot 2021-03-18 at 7 49 37 PM](https://user-images.githubusercontent.com/23396902/111712139-6d111100-8823-11eb-8bdf-97ea107f9cd5.png)
 I don't know how the decimal unit `kB` is written in French, but how do we let a translator know that they are supposed to translate as the`decimal unit` and not the `binary unit`?

Reply:
Yes. We could add more translation context later, when https://github.com/bitcoin/bitcoin/issues/21465 approach is implemented.